common/src/jni/main/include/compat.h

```diff
@@ -54,35 +54,9 @@ typedef long ssize_t;
 #define strcasecmp _stricmp
 
 // Windows doesn't define this either *sigh*...
-inline int vasprintf(char **ret, const char *format, va_list args) {
-    va_list copy;
-    va_copy(copy, args);
-    *ret = nullptr;
-
-    int count = vsnprintf(nullptr, 0, format, args);
-    if (count >= 0) {
-        char *buffer = static_cast<char *>(malloc((std::size_t)count + 1));
-        if (buffer == nullptr) {
-            count = -1;
-        } else if ((count = vsnprintf(buffer, static_cast<std::size_t>(count + 1), format, copy)) <
-                   0) {
-            free(buffer);
-        } else {
-            *ret = buffer;
-        }
-    }
-    va_end(copy);  // Each va_start() or va_copy() needs a va_end()
-
-    return count;
-}
+int vasprintf(char **ret, const char *format, va_list args);
```

Comment:
I think you need to revert this now that I just merged https://github.com/google/conscrypt/pull/38